### ISSUE TITLE
Setting up the conda environment

### ISSUE THREAD

**Issue by rxxxxxxb** [rxxxxxxb/NeRF-research#1]

**Comment by rxxxxxxb** [comment]
# This file may be used to create an environment using:
# $ conda create --name <env> --file <this file>
# platform: linux-64
_libgcc_mutex=0.1=main
_openmp_mutex=5.1=1_gnu
_tflow_select=2.1.0=gpu
absl-py=0.15.0=pyhd3eb1b0_0
aiohttp=3.8.3=py37h5eee18b_0
aiosignal=1.2.0=pyhd3eb1b0_0
appdirs=1.4.4=pyhd3eb1b0_0
astor=0.8.1=py37h06a4308_0
astunparse=1.6.3=py_0
async-timeout=4.0.2=py37h06a4308_0
asynctest=0.13.0=py_0
attrs=22.1.0=py37h06a4308_0
blas=1.0=mkl
blinker=1.4=py37h06a4308_0
brotli=1.0.9=h5eee18b_7
brotli-bin=1.0.9=h5eee18b_7
bzip2=1.0.8=h7b6447c_0
c-ares=1.18.1=h7f8727e_0
ca-certificates=2023.01.10=h06a4308_0
cachetools=4.2.2=pyhd3eb1b0_0
certifi=2022.12.7=py37h06a4308_0
cffi=1.15.1=py37h5eee18b_3
charset-normalizer=2.0.4=pyhd3eb1b0_0
click=8.0.3=pyhd3eb1b0_0
cryptography=39.0.1=py37h9ce1e76_0
cudatoolkit=10.0.130=0
cudnn=7.6.5=cuda10.0_0
cupti=10.0.130=0
cycler=0.11.0=pyhd3eb1b0_0
dataclasses=0.8=pyh6d0b6a4_7
dbus=1.13.18=hb2f20db_0
expat=2.4.9=h6a678d5_0
fftw=3.3.9=h27cfd23_1
flit-core=3.6.0=pyhd3eb1b0_0
fontconfig=2.14.1=h52c9d5c_1
fonttools=4.25.0=pyhd3eb1b0_0
freetype=2.12.1=h4a9f257_0
frozenlist=1.3.3=py37h5eee18b_0
gast=0.2.2=py37_0
giflib=5.2.1=h5eee18b_3
glib=2.69.1=he621ea3_2
google-auth=2.6.0=pyhd3eb1b0_0
google-auth-oauthlib=0.4.4=pyhd3eb1b0_0
google-pasta=0.2.0=pyhd3eb1b0_0
grpcio=1.42.0=py37hce63b2e_0
gst-plugins-base=1.14.1=h6a678d5_1
gstreamer=1.14.1=h5eee18b_1
h5py=2.10.0=py37hd6299e0_1
hdf5=1.10.6=h3ffc7dd_1
icu=58.2=he6710b0_3
idna=3.4=py37h06a4308_0
importlib_resources=5.2.0=pyhd3eb1b0_1
intel-openmp=2021.4.0=h06a4308_3561
jpeg=9e=h5eee18b_1
keras-applications=1.0.8=py_1
keras-preprocessing=1.1.2=pyhd3eb1b0_0
kiwisolver=1.4.4=py37h6a678d5_0
krb5=1.19.4=h568e23c_0
lcms2=2.12=h3be6417_0
ld_impl_linux-64=2.38=h1181459_1
lerc=3.0=h295c915_0
libbrotlicommon=1.0.9=h5eee18b_7
libbrotlidec=1.0.9=h5eee18b_7
libbrotlienc=1.0.9=h5eee18b_7
libclang=10.0.1=default_hb85057a_2
libdeflate=1.17=h5eee18b_0
libedit=3.1.20221030=h5eee18b_0
libevent=2.1.12=h8f2d780_0
libffi=3.4.2=h6a678d5_6
libgcc-ng=11.2.0=h1234567_1
libgfortran-ng=11.2.0=h00389a5_1
libgfortran5=11.2.0=h1234567_1
libgomp=11.2.0=h1234567_1
libllvm10=10.0.1=hbcb73fb_5
libpng=1.6.39=h5eee18b_0
libpq=12.9=h16c4e8d_3
libprotobuf=3.20.3=he621ea3_0
libstdcxx-ng=11.2.0=h1234567_1
libtiff=4.5.0=h6a678d5_2
libuuid=1.41.5=h5eee18b_0
libwebp=1.2.4=h11a3e52_1
libwebp-base=1.2.4=h5eee18b_1
libxcb=1.15=h7f8727e_0
libxkbcommon=1.0.1=hfa300c1_0
libxml2=2.9.14=h74e7548_0
libxslt=1.1.35=h4e12654_0
lz4-c=1.9.4=h6a678d5_0
markdown=3.1.1=py37_0
matplotlib=2.2.2=py37hb69df0a_2
mkl=2021.4.0=h06a4308_640
mkl-service=2.4.0=py37h7f8727e_0
mkl_fft=1.3.1=py37hd3c417c_0
mkl_random=1.2.2=py37h51133e4_0
multidict=6.0.2=py37h5eee18b_0
munkres=1.1.4=py_0
ncurses=6.4=h6a678d5_0
nspr=4.33=h295c915_0
nss=3.74=h0370c37_0
numpy=1.21.5=py37h6c91a56_3
numpy-base=1.21.5=py37ha15fc14_3
oauthlib=3.2.1=py37h06a4308_0
openssl=1.1.1t=h7f8727e_0
opt_einsum=3.3.0=pyhd3eb1b0_1
packaging=22.0=py37h06a4308_0
pcre=8.45=h295c915_0
pillow=9.4.0=py37h6a678d5_0
pip=22.3.1=py37h06a4308_0
ply=3.11=py37_0
pooch=1.4.0=pyhd3eb1b0_0
protobuf=3.20.3=py37h6a678d5_0
pyasn1=0.4.8=pyhd3eb1b0_0
pyasn1-modules=0.2.8=py_0
pycparser=2.21=pyhd3eb1b0_0
pyjwt=2.4.0=py37h06a4308_0
pyopenssl=23.0.0=py37h06a4308_0
pyparsing=3.0.9=py37h06a4308_0
pyqt=5.15.7=py37h6a678d5_1
pyqt5-sip=12.11.0=py37h6a678d5_1
pysocks=1.7.1=py37_1
python=3.7.16=h7a1cb2a_0
python-dateutil=2.8.2=pyhd3eb1b0_0
python-flatbuffers=2.0=pyhd3eb1b0_0
pytz=2022.7=py37h06a4308_0
qt-main=5.15.2=h327a75a_7
qt-webengine=5.15.9=hd2b0992_4
qtwebkit=5.212=h4eab89a_4
readline=8.2=h5eee18b_0
requests=2.28.1=py37h06a4308_0
requests-oauthlib=1.3.0=py_0
rsa=4.7.2=pyhd3eb1b0_1
scipy=1.7.3=py37h6c91a56_2
setuptools=65.6.3=py37h06a4308_0
sip=6.6.2=py37h6a678d5_0
six=1.16.0=pyhd3eb1b0_1
sqlite=3.40.1=h5082296_0
tensorboard=1.15.0=pyhb230dea_0
tensorboard-plugin-wit=1.8.1=py37h06a4308_0
tensorflow=1.15.0=gpu_py37h0f0df58_0
tensorflow-base=1.15.0=gpu_py37h9dcbed7_0
tensorflow-estimator=1.15.1=pyh2649769_0
tensorflow-gpu=1.15.0=h0d30ee6_0
termcolor=1.1.0=py37h06a4308_1
tk=8.6.12=h1ccaba5_0
toml=0.10.2=pyhd3eb1b0_0
tornado=6.2=py37h5eee18b_0
tqdm=4.64.1=py37h06a4308_0
typing-extensions=4.4.0=py37h06a4308_0
typing_extensions=4.4.0=py37h06a4308_0
tzdata=2022g=h04d1e81_0
urllib3=1.25.8=py37_0
webencodings=0.5.1=py37_1
werkzeug=0.16.1=py_0
wheel=0.38.4=py37h06a4308_0
wrapt=1.14.1=py37h5eee18b_0
xz=5.2.10=h5eee18b_1
yarl=1.8.1=py37h5eee18b_0
zipp=3.11.0=py37h06a4308_0
zlib=1.2.13=h5eee18b_0
zstd=1.5.2=ha4553b6_0
